admin_manual/ai/app_summarai.rst

```diff
@@ -0,0 +1,163 @@
+==========================================
+App: SummarAI (Talk chat summarize bot)
+==========================================
+
+.. _ai-app-summarai:
+
+The *SummarAI* app is one of the apps that utilize the llm or any other configured llm provider in Nextcloud and act as a configurable bot for `Nextcloud Talk`.
+The *SummarAI* app can be running on only open source models, on-premise or any configured llm provider and does so entirely on-premises.
+Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.
+
+The app currently supports the following languages:
+
+* English (en)
+
+As the models are entirely open source, the quality of translations may not be comparable to commercially available services.
+
+Requirements
+------------
+
+* Minimal Nextcloud version: 27
+* x86 CPU
+* Docker
+* AppAPI >= 2.3.2
+* Talk
+* Assistant
+* Local large language model
+
+Space usage
+~~~~~~~~~~~
+
+ * ~100MB
```

Comment:
this should just be a bullet point, no indentation (on your screenshot interpreted as quote)